_dashboards/discover/multi-data-sources.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: default
+title: Multiple data sources
+parent: Discover
+nav_order: 5
+---
+
+Multiple data sources is an experimental feature released in OpenSearch 2.4. We do not recommend using it in a production environment. For updates on the feature’s progress or to leave feedback on improving the feature, see the [multiple data sources git issue](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2448).
+{: .note }
+
+# Multiple data sources
+
+You can add multiple data sources to a single dashboard. OpenSearch Dashboards allows you to dynamically manage data sources, create index patterns based on those data sources, and execute queries against a specific data source and then combine visualizations in one dashboard.
+
+In this tutorial we provide the steps for enabling the data sources setting in Dashboards; adding credentials, data source connections, and index patterns; and combining visualizations in a single dashboard.
+
+## Try out multiple data sources in your local environment
+
+This tutorial uses a preconfigured data source and index pattern, and you aren’t required to configure settings. However, you’ll need to enable the multiple data sources setting in the configuration file before before getting started with exploring this feature.
+{: .note }
+
+Multiple data sources is an experimental feature that can't be deployed into production, but you can try it out with a sample data source and a sample index pattern. Before getting started, you must first edit the YAML configuration. The following section provides the steps to enable the multiple data sources feature.
+
+## Edit the YAML configuration to enable multiple data sources 
+
+Dashboards is configured in the cluster settings, and multiple data sources is disabled by default. To enable multiple data sources, you need to edit the configuration in `opensearch_dashboards.yml` and then restart the cluster. To enable multiple data sources:
```

Comment:
Do we want to style these features somehow – "multiple data sources"? "Discover" is bold in the first mention of it above. "Multiple data sources" is such a common string of words that alone it doesn't feel like a feature and gets lost in context, as in, "multiple data sources is disabled...". I know it's crossing our rule. But can we give them initial caps?

Reply:
I'm tracking with your comment. I revised throughout to "the multiple data sources feature" to clarify where appropriate (as in this para.)